HomeWork/MyHomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,6 @@
+import UIKit
+
+var greeting = "Hello, playground"
+let MyFirstName = "Vyacheslav"
```

Comment:
```suggestion
let MyFirstName = "Vyacheslav"
```
у нас используется камел кейс https://ru.wikipedia.org/wiki/CamelCase

причем первая буква всегда маленькая, если эт оне имя объекта.